manifest.json

```diff
@@ -214,6 +214,7 @@
                 "ObjectNotExists",
                 "PutObject",
                 "PutObjectAcl",
+                "PutBucketNotificationConfiguration",
```

Comment:
Should be sort alphabetically

Reply:
This element here is added automatically. 

Sorting it is nice, but it should be done by the generator, right?

Reply:
Yes, it should have been done automatically
https://github.com/async-aws/aws/blob/master/src/CodeGenerator/src/Command/GenerateCommand.php#L275-L278

My though is, @sunkan added it manually and ran `generate --all`